packages/strapi-admin/admin/src/containers/AuthPage/components/Register/index.js

```diff
@@ -0,0 +1,170 @@
+    <FormattedMessage id="Auth.privacy-policy-agreement.policy">
+      {content => (
+        <span
+          style={{ color: '#0097f7', cursor: 'pointer' }}
+          onClick={e => handleClick(e, 'privacy')}
+        >
+          {content}
+        </span>
+      )}
+    </FormattedMessage>
+  );
+
+  return (
+    <>
+      <Section textAlign="center">
+        <Logo />
+      </Section>
+      <Section withBackground>
+        <Padded top size="23px">
+          <Box errorMessage={get(requestError, 'errorMessage', null)}>
+            <form onSubmit={onSubmit}>
+              <InputWrapper>
+                <Input
+                  autoFocus
+                  error={formErrors.firstname}
+                  label="Auth.form.firstname.label"
+                  name="firstname"
+                  onChange={onChange}
+                  placeholder="Auth.form.firstname.placeholder"
+                  type="text"
+                  validations={{ required: true }}
+                  value={modifiedData.firstname}
+                />
+                <Input
+                  autoFocus
+                  error={formErrors.lastname}
+                  label="Auth.form.lastname.label"
+                  name="lastname"
+                  onChange={onChange}
+                  placeholder="Auth.form.lastname.placeholder"
+                  type="text"
+                  validations={{ required: true }}
+                  value={modifiedData.lastname}
+                />
+              </InputWrapper>
+              <Input
+                autoFocus
+                error={formErrors.email}
+                disabled={fieldsToDisable.includes('email')}
+                label="Auth.form.email.label"
+                name="email"
+                onChange={onChange}
+                placeholder="Auth.form.email.placeholder"
+                type="email"
+                validations={{ required: true }}
+                value={modifiedData.email}
+              />
+              <Input
+                error={formErrors.password}
+                label="Auth.form.password.label"
+                name="password"
+                onChange={onChange}
+                type="password"
+                validations={{ required: true }}
+                value={modifiedData.password}
+              />
+              <Input
+                error={formErrors.confirmPassword}
+                label="Auth.form.password.label"
+                name="confirmPassword"
+                onChange={onChange}
+                type="password"
+                validations={{ required: true }}
+                value={modifiedData.confirmPassword}
+              />
+
+              <Checkbox
+                message={() => (
+                  <CustomLabel id="Auth.form.register.news.label" values={{ terms, policy }} />
+                )}
+                name="news"
+                onChange={onChange}
+                value={modifiedData.news}
+              />
```

Comment:
This case is specific and the message prop is not appropriate. You can handle this case easily without the message props.
```javascript
<Flex alignItems="flex-start">
     <Checkbox name="news" onChange={onChange} value={modifiedData.news} />
     <Padded left size="xs" />
    <CustomLabel id="Auth.form.register.news.label" values={{ terms, policy }} /> // you have to add a line-height: normal to this component
</Flex>
```